packages/11ty/_plugins/figures/sequence/factory.js

```diff
@@ -13,11 +15,40 @@ module.exports = class SequenceFactory {
 
   /**
    * Iterates over a Figure's sequences and creates a sequence for each one
+   *
+   * @property {<Array[String]>} files  (Optional) An array of sequence image filenames (only used for passing through fixture data in tests)
    * @return {<Array[Sequence]>}
    */
-  create() {
+  create(files) {
     const { sequences } = this.figure.data
     if (!sequences) return
-    return sequences.map((sequence) => new Sequence(this.figure, sequence))
+    return sequences.map((sequence) => {
+      if (!files) {
+        files = getSequenceFiles(sequence, iiifConfig)
+      }
+      return new Sequence(this.figure, sequence, files)
+    })
+  }
+
+  /**
+   * Read the directory where sequence images exist to create an array of filenames
+   *
+   * @param  {Object} figure     Figure data
+   * @param  {Object} iiifConfig IIIF Config data
+   * @return {Array<string>}     An array of filenames
+   */
+  getSequenceFiles(sequence, iiifConfig) {
+    if (!sequence) return
+    const { dirs } = iiifConfig
+    const { imagesDir, inputRoot } = dirs
+    const sequenceDir = path.join(inputRoot, imagesDir, sequence.id)
+    const defaultSequenceRegex = /^\d{3}\.(jpg|png)$/
+    const sequenceRegex = sequence.regex
+      ? new RegExp(sequence.regex.slice(1, -1))
+      : defaultSequenceRegex
+    if (!fs.existsSync(sequenceDir)) return
+    return fs
+      .readdirSync(sequenceDir)
+      .filter((sequenceItemFilename) => sequenceItemFilename.match(sequenceRegex))
   }
```

Comment:
@anderspollack since the `iiifConfig` is passed to the `sequenceFactory` as an argument it probably not needed as an argument to `getSequenceFiles`, a few additional comments to this method inline below—

```suggestion
  getSequenceFiles(sequence) {
    if (!sequence) return
    
    const sequenceDir = path.join(inputRoot, imagesDir, sequence.id)
    
    /**
     * Default file name pattern for 360° images,
     * zero padded three digit degrees
     * @todo refactor to use default pattern in publication configuration
     */
    const defaultPattern = /^\d{3}\.(jpg|png)$/
    
    const sequenceRegex = sequence.regex
      ? new RegExp(sequence.regex.slice(1, -1))
      : defaultPattern
    
    // @todo log a warning or perhaps throw an error?
    if (!fs.existsSync(sequenceDir)) return
    
    return fs
      .readdirSync(sequenceDir)
      .filter((filename) => filename.match(sequenceRegex))
  }
```